docs/source/03_plugin_structure.rst

```diff
@@ -0,0 +1,44 @@
+.. _plugin_structure-section:
+
+Plugins Structure
```

Comment:
In general, this section is fine. I would ask you just to use the `code` style in the commands like 
"... invoke the `compile` command to generate ..." in the _alfasim-sdk package_ . (this entire section does not use `code` style, but the next sections use it)